### DISCUSSION TITLE
racket-langserver

### DISCUSSION THREAD

**Discussion by jgarte** [joaotavora/eglot#694] [Ideas]
Has anyone tested [racket-langserver](https://github.com/jeapostrophe/racket-langserver) with eglot yet?

**Comment by skangas** [comment]
I have now, it works well with:
```
(add-to-list 'eglot-server-programs '(racket-mode . ("racket" "-l" "racket-langserver")))
```

**Reply by skangas** [reply]
I've added this to `eglot-server-programs` in https://github.com/joaotavora/eglot/commit/86221e05c0909645761dc72e917888b8a782eeab.
